internal/plugin/connectors/http/conjur/plugin.go

```diff
@@ -16,16 +18,26 @@ func PluginInfo() map[string]string {
 	}
 }
 
-// NewConnector returns an http.Connector that decorates each incoming http
-// request with a basic auth header.
-func NewConnector(conRes connector.Resources) http.Connector {
-	return &Connector{
-		logger: conRes.Logger(),
-	}
-}
-
 // GetHTTPPlugin is required as part of the Secretless plugin spec for HTTP
 // connector plugins. It returns the HTTP plugin.
 func GetHTTPPlugin() http.Plugin {
-	return http.NewConnectorFunc(NewConnector)
+	newConnector, err := generichttp.NewConnectorConstructor(
+		&generichttp.ConfigYAML{
+			CredentialValidations: map[string]string{
+				// Unsure
```

Comment:
are you looking for the standard format of the Conjur token? that's probably worth asking about in Slack and adding here before this gets merged

Reply:
Good catch - I checked with Jonah yesterday and forgot to remove it.

Reply:
if it helps it looks like it's just a JWT: https://docs.conjur.org/Latest/en/Content/Get%20Started/cryptography.html?Highlight=cryptography#Authenticationtokens

Reply:
whoops, missed your comment. 👍